common/src/main/java/io/netty/util/HashedWheelTimer.java

```diff
@@ -558,6 +600,8 @@ void remove() {
             HashedWheelBucket bucket = this.bucket;
             if (bucket != null) {
                 bucket.remove(this);
+            } else {
```

Comment:
Actually, I would still prefer to have pendingTimeout counter maintained so that clients of HashWheel could use the public pendingTimeoutsCount method for monitoring/their own guards.